src/patterns/molecules/heading/heading.scss

```diff
@@ -0,0 +1,82 @@
+@use "sass:list";
+@use "sass:map";
+@use "../../../core-styling/settings/baselinegrid";
+@use "../../../core-styling/settings/font";
+@use "../../../core-styling/mixins/typography";
+@use "../../../core-styling/mixins/scale";
+@use "../../../core-styling/mixins/validation";
+
+$_h1-line-height: baselinegrid.$space-medium;
```

Comment:
Alternative would be to make these configurable with an implementation like e.g.: 
```
$h1-line-height: baselinegrid.$space-medium !default;
```
However this is outside the scope of configurability we're allowing, so will keep them as private, and soley defined by the baseline grid.